Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/YoutubeQualityScript.js

```diff
@@ -19,13 +19,16 @@ window.__firefox__.includeOnce("YoutubeQuality", function($) {
     return document.getElementById('movie_player') || document.querySelector('.html5-video-player');
   }
   
-  function hasAPIs(player) {
-    if (!player) {
-      return false;
+  // Returns -1 if the api does not exist.
+  // If it does exist it returns number of available video qualities of the player.
+  function hasAPIsAndEnoughVideoQualities(player) {
+    if (!player || typeof player.getAvailableQualityLevels === 'undefined') {
+      return -1;
     }
     
-    return typeof player.getAvailableQualityLevels !== 'undefined';
+    return player.getAvailableQualityLevels().length;
   }
```

Comment:
This is implied to be a boolean op, so best to return true/false, so return false instead of -1 and then return `.length > 0`. Then you can remove the > 0 check below